docs/supplements/OpenScience.md

```diff
@@ -59,3 +60,25 @@ Daniel Méndez, Daniel Graziotin, Stefan Wagner, and Heidi Seibold. 2019. Open s
 GitHub. 2016. Making Your Code Citable. Retrieved July 12, 2020 from [https://guides.github.com/activities/citable-code/](https://guides.github.com/activities/citable-code/). (How to automatically archive a GitHub repository to Zenodo)
 
 Figshare. How to connect Figshare with your GitHub account. Retrieved July 12, 2020 from [https://knowledge.figshare.com/articles/item/how-to-connect-figshare-with-your-github-account](https://knowledge.figshare.com/articles/item/how-to-connect-figshare-with-your-github-account) (How to automatically archive a GitHub repository to Figshare)
+
+## Examplar Artifacts
+Below are example artifacts for different types of software engineering research to aid in preparing reusable and functional research artifacts.
+
+#### Method: Experiments (System Evaluations, etc) and Quasi-Experiments
```

Comment:
The headings here should match the standards headings.

https://www2.sigsoft.org/EmpiricalStandards/docs/standards